From 472c4f941743be96428d3b4c612c465c3c81b32e Mon Sep 17 00:00:00 2001
From: Atchaya R <ratchaya@qti.qualcomm.com>
Date: Fri, 20 Feb 2026 16:55:05 +0530
Subject: [PATCH] prpl-patches: add support for ChannelsInUse DM to be
 read-only

Since this DM should be updated based on the channel and
OperatingStandardFrequency set, it should not take input from user.

Change-Id: I299356e050b13512fd5e60e86b4907217a46d43a
Signed-off-by: Atchaya R <ratchaya@qti.qualcomm.com>
---
 .../0008-add-support-ChannelsInUse-dm-read-only       | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 package/feeds/feed_prplmesh/pwhm/patches/0008-add-support-ChannelsInUse-dm-read-only


--- a/odl/wld_radio.odl
+++ b/odl/wld_radio.odl
@@ -725,7 +725,7 @@ define probelist_t{
 		 * currently in use (including any that it is using itself).
 		 * Ranges in the form "n-m" are permitted. </p>
 		 */
-		string ChannelsInUse;
+		%read-only string ChannelsInUse;
 
 		/**
 		 * <p>Indicates whether automatic channel selection is supported
